css/components/progress-bar.css

```diff
@@ -2,6 +2,7 @@
     display: flex;
     flex-direction: column;
     gap: 10px;
+    padding-bottom: 1em;
 }
```

Comment:
progress-bar text `p` should be 14px.

Reply:
> progress-bar text `p` should be 14px.

I check original - there text size - p => 18 px (always).